docker/build.sh

```diff
@@ -46,7 +46,7 @@ main() {
 }
 
 GetGcc() {
-  wget -q https://developer.arm.com/-/media/Files/downloads/gnu-rm/$GCC_ARM_VER/$GCC_ARM_PATH-$MACHINE-linux.tar.bz2 -O - | tar -xj -C $TOOLS_DIR/
+  wget -q https://developer.arm.com/-/media/Files/downloads/gnu/$GCC_ARM_VER/binrel/arm-gnu-toolchain-$GCC_ARM_VER-$MACHINE-arm-none-eabi.tar.xz -O - | tar -xJ -C $TOOLS_DIR/
```

Comment:
It would be better to use the `GCC_ARM_PATH` here.

```suggestion
  wget -q https://developer.arm.com/-/media/Files/downloads/gnu/$GCC_ARM_VER/binrel/$GCC_ARM_PATH.tar.xz -O - | tar -xJ -C $TOOLS_DIR/
```